wire-ios-sync-engine/Source/Synchronization/Strategies/RegistrationStrategy.swift

```diff
@@ -60,7 +60,7 @@ extension RegistrationStrategy: ZMSingleRequestTranscoder {
                 userInfoParser?.upgradeToAuthenticatedSession(with: $0)
             }
             registrationStatus.success()
-        } else {
+        } else { // TODO: handle new errors? 400: "invalid-domain" and 403: "condition-failed"
```

Comment:
If we don't specifically handle these new errors, does the registration status still consider it an error? The login and registration flows are being re-written so probably doesn't make much sense to handle these errors if not absolutely necessary.

Reply:
removed TODO